<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Olaf Chojnacki</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kalkulator naukowy</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;nic&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_wprFun">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wprowadź funkcję:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>515</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="boxLayo_Buttons">
     <item>
      <widget class="QPushButton" name="Btn_Calka">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Halko&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Całka</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Pochodna">
       <property name="text">
        <string>Pochodna</string>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Wykres">
       <property name="text">
        <string>Wykres</string>
       </property>
       <property name="shortcut">
        <string>Alt+W</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Miejsca_zerowe">
       <property name="text">
        <string>Miejsca zerowe</string>
       </property>
       <property name="shortcut">
        <string>Alt+M</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="edt_funkcja">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tekst funkcji&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>106</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lab_wprFun_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>po:  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_funkcjaPo">
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>199</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lab_wprFun_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>od:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edt_od">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="lab_wprFun_4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>do:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_do">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>193</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="lab_wprFun_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Jeżeli całka oznaczona</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbox_ozn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionZamknij"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>edt_funkcja</tabstop>
  <tabstop>edt_funkcjaPo</tabstop>
  <tabstop>cbox_ozn</tabstop>
  <tabstop>edt_od</tabstop>
  <tabstop>edt_do</tabstop>
  <tabstop>Btn_Calka</tabstop>
  <tabstop>Btn_Pochodna</tabstop>
  <tabstop>Btn_Wykres</tabstop>
  <tabstop>Btn_Miejsca_zerowe</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_Calka</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fun_calka()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>10</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Pochodna</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fun_pochodna()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Wykres</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fun_wykres()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_Miejsca_zerowe</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>fun_mscZerowe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbox_ozn</sender>
   <signal>toggled(bool)</signal>
   <receiver>edt_do</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>489</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbox_ozn</sender>
   <signal>toggled(bool)</signal>
   <receiver>edt_od</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>odczytFunkcji()</slot>
  <slot>fun_calka()</slot>
  <slot>fun_pochodna()</slot>
  <slot>fun_wykres()</slot>
  <slot>fun_wynikRownania()</slot>
  <slot>fun_mscZerowe()</slot>
 </slots>
</ui>
